apps/editors/views.py

```diff
@@ -481,6 +481,7 @@ def queue_fast_track(request):
 @addons_reviewer_required
 def queue_moderated(request):
     rf = (Review.objects.exclude(Q(addon__isnull=True) |
+                                 Q(addon__is_listed=False) |
```

Comment:
Looks good, but we don't really need a `Q` object here I believe, it could be written:
```python
.exclude(addon__isnull=True)
.exclude(addon__is_listed=False)
.exclude(reviewflag__is_null=True)
```